profile/heavy_weight_benchmark.rb

```diff
@@ -0,0 +1,212 @@
+$LOAD_PATH.unshift(File.dirname(__FILE__))
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), "..", "lib"))
+
+require 'mongo'
+require 'benchmark'
+require_relative 'benchmark_helper'
+require 'thread'
+
+##
+# Perform 'heavyweight' benchmarks. This includes
+#
+# LDJSON multi-file import
+# LDJSON multi-file export
+# GridFS multi-file upload
+# GridFS multi-file download
+#
+# The heavyweight benchmark is intended to test concurrency performance.
+#
+##
+def heavyweight_benchmark!
+  #bench_helper = BenchmarkHelper.new('perftest','corpus', 200)
+  bench_helper = BenchmarkHelper.new('foo','bar', 200)
+  database = bench_helper.database
+  collection = bench_helper.collection
+
+
+
+  ##
+  # LDJSON multi-file import
+  #
+  # Dataset: LDJSON_MULTI
+  #
+  # - Drop the 'perftest' database.
+  # - Construct whatever objects, threads, etc. are required for importing the dataset but
+  #   do not read any data from disk.
+  #
+  # Measure: Do an unordered insert of all 1,000,000 documents in the dataset into the 'corpus'
+  #          collection as fast as possible.  Data must be loaded from disk during this phase.
+  #          Concurrency is encouraged.
+  #
+  # - Drop the 'perftest' database
+  ##
+  database.drop
+
+  # The directory name, or path to the directory, in which the LDJSON files are expected to be
+  ldjson_data_files_directory = "LDJSON_data_file_directory"
+  # Array of expected file names
+  ldjson_multi_files = Array.new(100) {|i| "LDJSON%02d.txt" % (i+1) }
```

Comment:
The spec uses "gridfstest%03d". If I do "LDJSON#{i+1}.txt", it'll be different. Doesn't really matter either way: should it be overruled?